### ISSUE TITLE
AlertDialog Accessibility Violation

### ISSUE THREAD

**Issue by SashaBajzek** [tamagui/tamagui#3316]
### Current Behavior

Using VoiceOver on a Mac in Chrome:

When I activate the "Show Alert" button in this page: https://tamagui.dev/ui/alert-dialog, the VoiceOver cursor is not moved inside the AlertDialog, and is instead left behind on the page, unable to get into the dialog.

### Expected Behavior

Ideally, the rest of the page should be [inert](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert), and the VoiceOver cursor should be moved the heading of the dialog with the index set to -1 so the heading can't be tabbed to again.

### Tamagui Version

```markdown
https://tamagui.dev/ui/alert-dialog
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
Using VoiceOver on a Mac in Chrome:

When I activate the "Show Alert" button in this page: https://tamagui.dev/ui/alert-dialog, the VoiceOver cursor is not moved inside the AlertDialog, and is instead left behind on the page, unable to get into the dialog.
```

### System Info

```markdown

```
